### PR TITLE
feat: add go-zero example for uptrace reporting

### PR DESCRIPTION
**Pull request by toby1991**
fix: https://github.com/uptrace/uptrace/issues/212

related:
* https://github.com/zeromicro/go-zero/issues/3218
* https://github.com/zeromicro/go-zero/pull/3219

### CODE REVIEW

File: example/go-zero/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  Name: sdkrpc.rpc
+  Endpoint: uptrace.apm:14317
+  Sampler: 1.0
+  Batcher: otlpgrpc
+  OtlpHeaders:
+    uptrace-dsn: http://project2_secret_token@uptrace.apm:14317/2
+
+```
+
+> https://uptrace.dev/get/ingest/opentelemetry.html#already-using-opentelemetry-sdk
```
Comment (vmihailenco) [inline example/go-zero/README.md:17]: Thanks! I will also add an entry about go-zero to https://uptrace.dev/opentelemetry/instrumentations/

But could we make this example runnable? E.g. we could change this example to use Uptrace from [Docker](https://github.com/uptrace/uptrace/tree/master/example/docker) so users can run this with few commands.

We also need some way to trigger a request, for example, using a browser (open some URL) or by providing a `client.go` that makes some requests.

What do you think?

Comment (toby1991) [reply example/go-zero/README.md]: Done~

Comment (toby1991) [reply example/go-zero/README.md]: image

**Comment by vmihailenco** [comment]
Thanks!

**Pull request merged**